page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/PageManager.java

```diff
@@ -175,27 +165,96 @@ public Page getPage(String packageName, String title) throws PageInitializationE
     }
 
     /**
-     * Run constructor of specified page class and put its instance into static
-     * {@link PageContext#currentPage} variable
+     * Redirect to Page by Page Entry url value
      *
-     * @param page page class
-     * @return initialized page
-     * @throws PageInitializationException if failed to execute corresponding
-     * page constructor
+     * @param title a {@link java.lang.String} object.
+     * @return a Page object.
+     * @throws PageInitializationException TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)